src/lib/server/blog/posts.ts

```diff
@@ -120,3 +127,51 @@ export function getOfficerByGhUsername(ghUsername: string): Officer | null {
   );
   return officer ?? null;
 }
+
+/**
+ * Performs the following post-processing steps on the HTML content of a
+ * blog post
+ * 1. Removes the "disabled" attribute from any checkboxes in the document
+ * 2. Wraps all .task-list-item contents to make them css selectable
+ * @param html The HTML content of the blog post
+ * @returns The modified HTML string
+ */
+function postProcessHTML(html: string) {
+  const $ = cheerio.load(html, { xmlMode: true });
+  // remove disabled attribute from checkboxes
+  $('input').removeAttr('disabled');
```

Comment:
Specify that the only input elements we want to manipulate are checkboxes. 
```suggestion
  $('input[type=checkbox]').removeAttr('disabled');
```

Reply:
👌